build_info/kimg4.control

```diff
@@ -0,0 +1,12 @@
+Package: kimg4
+Version: @DEB_KIMG4_V@
+Architecture: all
+Maintainer: @DEB_MAINTAINER@
+Author: kritanta
+Section: Python
+Depends: pyaes
+Priority: optional
+Homepage: https://github.com/ricmoo/pyaes
```

Comment:
Wrong homepage.

Reply:
Link the right one, then

Reply:
https://github.com/cxnder/kimg4

Reply:
(I legit thought I had the right one; I thought this was for pyaes)